db/migrate/20150824134751_add_user_to_order.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserToOrder < ActiveRecord::Migration
+  def change
+    change_table :orders do |t|
+      t.belongs_to :seller
```

Comment:
in order to make this work, you should help rails figure out which table are they accessing, since it will create a `seller_id` column referencing the  `sellers` table (that doesn't exist).

From: http://edgeguides.rubyonrails.org/active_record_migrations.html#foreign-keys

```
The options hash can include the following keys:

:column ==> The foreign key column name on from_table. Defaults to to_table.singularize + "_id"
:primary_key ==> The primary key column name on to_table. Defaults to id.
```